builtin/providers/aws/resource_aws_kinesis_analytics.go

```diff
@@ -0,0 +1,541 @@
+package aws
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/kinesisanalytics"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func shim(*schema.ResourceData, interface{}) error {
+	return errors.New("func UPDATE Not implemented in resource_aws_kinesis_analytics.go")
+}
+
+func resourceAwsKinesisAnalytics() *schema.Resource {
+	return &schema.Resource{
+
+		Create: resourceAwsKinesisAnalyticsCreate,
+		Read:   resourceAwsKinesisAnalyticsRead,
+		Update: shim,
```

Comment:
You don't need the shim here - if you don't want to use Update, set `ForceNew: true` on all params